docs/migration/migrate_7_9.asciidoc

```diff
@@ -0,0 +1,736 @@
+
+```ts
+const encryptedSavedObjectsClient = plugins.encryptedSavedObjects.getClient(['hiddenType']);
+return encryptedSavedObjectsClient.getDecryptedAsInternalUser('hiddenType',  '123',   { namespace: 'some-namespace' });
+```
+
+*via https://github.com/elastic/kibana/pull/66879[#66879]*
+
+====
+
+[[breaking_79_alerting_api]]
+.The `alerting` plugin was renamed `alerts` to follow the {kib} styleguide
+[%collapsible]
+====
+
+This includes the following API changes:
+
+* Changed actions `BASE_ALERT_API_PATH` to ` '/api/alerts'` because according to the styleguide, it should keep the structure `/api/plugin_id`
+* Changed endpoint `/api/alert/_find` just to `/api/alerts/_find`
+* Changed `/types` to `/list_alert_types`
+* Changed POST `/api/alert` to POST `/api/alerts/alert`
+* Changed GET `/api/alert/{id}` to GET `/api/alerts/alert/{id}`
+* Changed PUT  `/api/alert/{id}` to PUT `/api/alerts/alert/{id}`
+* Changed DELETE `/api/alert/{id}` to DELETE `/api/alerts/alert/{id}`
+* Changed GET  `/api/alert/{id}/state` to GET `/api/alerts/alert/{id}/state`
+* Changed POST `/api/alert/{id}/_enable` to POST `/api/alerts/alert/{id}/_enable`
+* Changed POST `/api/alert/{id}/_disable` to POST `/api/alerts/alert/{id}/_disable`
+* Changed POST `/api/alert/{id}/_mute_all` to POST `/api/alerts/alert/{id}/_mute_all`
+* Changed POST `/api/alert/{alertId}/alert_instance/{alertInstanceId}/_mute` to POST `/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_mute`
+* Changed POST `/api/alert/{id}/_unmute_all` to POST `/api/alerts/alert/{id}/_unmute_all`
+* Changed POST `/api/alert/{id}/_update_api_key` to POST `/api/alerts/alert/{id}/_update_api_key`
+* Changed POST `/api/alert/{alertId}/alert_instance/{alertInstanceId}/_unmute` to POST `/api/alerts/alert/{alertId}/alert_instance/{alertInstanceId}/_unmute`
+
+*via https://github.com/elastic/kibana/pull/66838[#66838]*
+
+
+====
+
+[[breaking_79_new_management_api]]
+.The new platform API is now implemented in Management
+[%collapsible]
+====
+
+This change:
+
+* Refactors out use of `registerLegacyApp` and uses react-router-dom for routing.
+* Implements a landing page and sidebar in the Management plugin.
+* Removes the legacy API from `src/plugins/management/public/plugin.ts` and related code.
+
+*via https://github.com/elastic/kibana/pull/66781[#66781]*
+
+====
+
+[[breaking_79_alert_hidden]]
+.The Alerting SavedObject type `alert` is now a hidden type
+[%collapsible]
+====
+
+Interaction with the Alerting SavedObject type requires you
+to tell your `SavedObjectsClient` to include the `alert` hidden
+type as follows:
+
+``` ts
+core.savedObjects.getScopedClient(request, { includedHiddenTypes: ['alert'] })
+```
+
+Do not circumvent the authorization model by accessing these objects directly.
+Use AlertsClient instead.
+
+*via https://github.com/elastic/kibana/pull/66719[#66719]*
+
+====
+
+[[breaking_79_oss_features]]
+.OSS features registration moved to Kibana platform
```

Comment:
@lcawl Is "OSS" the correct terminology here?

Reply:
The style guide refers to "open source features", so I think that's clearer